package.json

```
@@ -12,7 +12,7 @@
     "docker:up": "docker compose -f docker-compose.dev.yml up -d && sleep 5",
     "docker:nuke": "docker compose -f docker-compose.dev.yml down --volumes --remove-orphans",
     "lint": "turbo run lint",
-    "dev": "pnpm docker:up && NEXT_PUBLIC_E2E_TEST=false turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
+    "dev": "pnpm docker:up && cross-env NEXT_PUBLIC_E2E_TEST=false && turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
```

```suggestion
    "dev": "pnpm docker:up && cross-env NEXT_PUBLIC_E2E_TEST=false turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
```

It should work as expected without chaining a `&`

---

done, removed `&&`